test/lsVersion.ts

```diff
@@ -0,0 +1,73 @@
+import { StatusBar, VSBrowser, Workbench } from "vscode-extension-tester";
+import { expect } from "chai";
+import * as path from "path";
+import { getStatusBarItem } from "./getStatusBarItem";
+import { homedir } from "os";
+
+describe("LS version", function () {
```

Comment:
Naming is pretty inconsistent in this PR. one time you're talking about the LS version, another you're talking about Scarb picking.

I think we're more interested in testing for CairoLS lookup rather than Scarb only. Scarb lookup is more like a detail of the boarder process. Please rename things appropriately.